govgen-1/gentx/gentx-allnodes.json

```diff
@@ -0,0 +1 @@
+{"body":{"messages":[{"@type":"/cosmos.staking.v1beta1.MsgCreateValidator","description":{"moniker":"Allnodes","identity":"3999DA33020A4DBC","website":"https://www.allnodes.com","security_contact":"support@allnodes.com","details":"Reliable non-custodial Validator run by the industry leader - Allnodes."},"commission":{"rate":"0.000000000000000000","max_rate":"1.000000000000000000","max_change_rate":"0.010000000000000000"},"min_self_delegation":"1","delegator_address":"govgen1e4jpknx6p3v2skfpcz63n3z3vemnhhvh3eqrvs","validator_address":"govgenvaloper1e4jpknx6p3v2skfpcz63n3z3vemnhhvhqmfzx2","pubkey":{"@type":"/cosmos.crypto.ed25519.PubKey","key":"AR9j8R3dE/8ALo5D4NTHnU8LrGBrCrXqU56obyaL1O4="},"value":{"denom":"ugovgen","amount":"1000000"}}],"memo":"834b9dacfa12f85b9b841f887d04ae55ade2482c@172.17.0.20:26656","timeout_height":"0","extension_options":[],"non_critical_extension_options":[]},"auth_info":{"signer_infos":[{"public_key":{"@type":"/cosmos.crypto.secp256k1.PubKey","key":"Aq4+iJFdVNHn6AityuyFuxS2pNUiYHalPqXK9CJaqDtJ"},"mode_info":{"single":{"mode":"SIGN_MODE_DIRECT"}},"sequence":"0"}],"fee":{"amount":[],"gas_limit":"200000","payer":"","granter":""}},"signatures":["Fah2AvOou0msODPs5aKluUN8XZb+NQO6UMeH17kzpj0F4gdcDpdmc/TkF2xS8twAevAJWKXyRS3hhhTRnskRyw=="]}
```

Comment:
Please update with `max_rate` and `max_change_rate` to 0

Reply:
Done